File: platforms/iPad/Accounter iOS 1.0.2/Accounter/Cell.m
//
//  Cell.m
//  Accounter iOS2
//
//  Created by Amrit Mishra on 11/11/11.
//  Copyright 2011 __MyCompanyName__. All rights reserved.
//

#import "Cell.h"


@implementation Cell

- (void)setValue:(NSString *)stringValue{
    value = stringValue;
}

- (NSString *)getValue{
    return value;
}

- (void)setName:(NSString *)stringName{
    name = stringName;
}

- (NSString *)getName{
    return name;
}


@end

File: platforms/blackberry/Accounter/src/com/vimukti/accounterbb/result/Cell.java
package com.vimukti.accounterbb.result;

public class Cell {
	private String title;
	private String value;

	public void setValue(String value) {
		this.value = value;
	}

	public String getValue() {
		return value;
	}

	public void setTitle(String name) {
		this.title = name;
	}

	public String getTitle() {
		return title;
	}
}
